mobile/lib/models/place_details.dart

```diff
@@ -105,8 +95,13 @@ class PlaceDetails {
   }
 
   static PlaceDetails siteToPLace(Site site) {
-    return PlaceDetails(site.getName(), site.getLocation(), site.id,
-        const Uuid().v4(), site.latitude, site.longitude);
+    return PlaceDetails(
+        name: site.name,
+        location: site.location,
+        siteId: site.id,
+        placeId: const Uuid().v4(),
```

Comment:
How is this placeId utilised in the app?c @NoahNsimbe

Reply:
It's the Id of a place searched using google maps